install-deps.sh

```diff
@@ -21,7 +21,7 @@ export LC_ALL=C # the following is vulnerable to i18n
 
 function ensure_decent_gcc_on_deb {
     # point gcc to the one offered by distro if the used one is different
-    local old=$(readlink -e /usr/bin/x86_64-linux-gnu-gcc | cut -d'-' -f2)
+    local old=$(readlink -e /usr/bin/gcc | cut -d'-' -f2)
```

Comment:
I don't have Xenial or Trusty machines handy to look, but it seems like we are now looking for a symlink that these two Ubuntu major versions have in common. Maybe that is `/usr/bin/gcc-linux` ?

If there is no common symlink between these two, maybe you could just pass the filename as a second argument to the function? And then do something like:

```
local old=$(readlink -e $gcc_symlink | cut -d'-' -f2)
```

Reply:
@smithfarm i think i messed up the building machine. yes, there is a `/usr/bin/gcc-linux`. and i am sure that's not a file shipped by ubuntu. guess i managed to mess it up. will try to fix it.